cuqi/distribution/_cauchy.py

```diff
@@ -34,6 +34,15 @@ class Cauchy(Distribution):
         x = cuqi.distribution.Cauchy(location, scale)
         x.logpdf(0) # -1.1447298858494002
 
+    .. code-block:: python
+
+        # % Generate a multivariate Cauchy distribution
+        import cuqi
+        location = [0, 0]
+        scale  = [1, 1]
```

Comment:
Can we make a slightly less trivial one please :)  Where location and scale are different, so that it is clear that is it not just an iid version.

Reply:
Good suggestion :)